app/src/controllers/appInfo/selectors.js

```diff
@@ -15,4 +15,4 @@ export const analyticsEnabledSelector = (state) =>
   extensionConfigSelector(state)[ANALYTICS_ALL_KEY] === 'true';
 export const analyzerExtensionsSelector = (state) => extensionsSelector(state).analyzer || [];
 
-export const authExtensionsSelector = (state) => uatInfoSelector(state).auth_extensions || {};
```

Comment:
Please make the `uatInfoSelector = compositeInfoSelector(state).uat || {};` and change `authExtensionsSelector = uatInfoSelector(state).auth_extensions || {}`